test/transcoding/builtin_calls.ll

```diff
@@ -0,0 +1,36 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
We actually have a couple (or more) tests related to built-in variables. For example https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/master/test/transcoding/builtin_vars_arithmetics.ll . Could you please also check that round trip translation (i.e. LLVM-SPV-IR-LLVM) now has been fixed and works for these tests?

Reply:
Thank you.